### ISSUE TITLE
[AWS] Introduce ignore_older & start_timestamp

### ISSUE THREAD

**Issue by Kavindu-Dodan** [elastic/integrations#11919]
**Background**

This issue is to track and upgrade AWS log integrations that utilize filebeat with `ignore_older` & `start_timestamp` introduced through https://github.com/elastic/beats/pull/41817. 

With the mentioned properties, users can control S3 (or S3 compatible) polling based log ingestion and improve filebeat registry memory usage.

**AC**

 Once https://github.com/elastic/beats/pull/41817 is merged, expose the above-mentioned properties through relevant integrations. 

Note - Configurations will be released with beats `8.18.0`

**Comment by lucabelluccini** [comment]
Hello @Kavindu-Dodan 

Thank you for the work on this enhancement already on Beats.

Please try to reach out integration developers who use the `aws-s3` input (with polling mode) in their integrations to notify them about the new settings available in 8.18.0

One key integration which would possibly benefit from this is https://www.elastic.co/guide/en/integrations/current/cloudflare_logpush.html#cloudflare_logpush-to-collect-data-from-s3-compatible-cloudflare-r2-buckets-follow-the-below-steps

**Comment by Kavindu-Dodan** [comment]
@lucabelluccini Good point. New configurations introduced at Beats are related to any integration that use S3 input for data ingestion. Most of the related integrations belong to my team. And I have added the team label that maintains the Cloudflare logs integrations.
